### ISSUE TITLE
Inconsistent name of keyword argument of fetchOrderBook() between exchanges in Python version

### ISSUE THREAD

**Issue by dhmk815** [ccxt-dev/ccxt#365]
Hello, I found something inconsistent in the name of keyword argument of `fetchOrderBook()`.

```
import inspect

argspec = inspect.getfullargspec(ccxt.bithumb().fetchOrderBook)
print(argspec.args)
### ['self', 'symbol', 'params']

argspec = inspect.getfullargspec(ccxt.bittrex().fetchOrderBook)
print(argspec.args)
### ['self', 'market', 'params']

argspec = inspect.getfullargspec(ccxt.poloniex().fetchOrderBook)
print(argspec.args)
### ['self', 'market', 'params']
```

As shown in above code, in bithumb exchange, the keyword for `fetchOrderBook()` is `symbol`.
But, in other exchanges, it's `market`.
Of course, it's not a big deal if I don't put keyword when calling the function.

**Comment by kroitor** [comment]
Hi, @dhmk815 ! Thanks for pointing that out, we will bring it into order, will upload the edits asap!

**Comment by dhmk815** [comment]
Hi, @kroitor ! Thank you for your reply.

**Comment by kroitor** [comment]
This issue is fixed as of version 1.9.223+. Thx again for your feedback!
